src/appkit.cs

```diff
@@ -11694,7 +12066,7 @@ interface NSProgressIndicator : NSAccessibilityProgressIndicator {
 	}
 
 	[BaseType (typeof (NSObject))]
-	partial interface NSResponder : NSCoding {
+	partial interface NSResponder : NSCoding, NSTouchBarProvider {
 		[Export ("tryToPerform:with:")]
 		bool TryToPerformwith (Selector anAction, [NullAllowed] NSObject anObject);
```

Comment:
@chamons there's must be a spell check API on macOS, we already have the test for iOS it just need a mac-specific implementation (as the API used is only available on iOS and derivatives).